src/controllers/confirmRules.controller.js

```diff
@@ -24,19 +46,23 @@ module.exports = class ConfirmRulesController extends BaseController {
       return ConfirmRulesController.doGet(request, reply, errors)
     } else {
       const authToken = CookieService.getAuthToken(request)
-      const applicationLineId = CookieService.getApplicationLineId(request)
+      try {
+        const applicationLineId = CookieService.getApplicationLineId(request)
 
-      // Get the Site for this application (if we have one)
-      const confirmRules = new ConfirmRules({
-        applicationLineId: applicationLineId
-      })
+        const complete = await this.isComplete(request)
+        if (complete) {
+          return reply.redirect(Constants.Routes.TASK_LIST.path)
+        }
 
-      try {
-        await confirmRules.save(authToken)
-        return reply.redirect(Constants.Routes.TASK_LIST.path)
+        const confirmRules = new ConfirmRules({
```

Comment:
I'd prefer to use non-ES6 syntax here to make it more maintainable by less experienced Javascript developers. 

In addition we are missing some properties here, we need applicationId and complete to be specified as well in the call to the constructor.

So it should be:

        const confirmRules = new ConfirmRules({
          applicationId: applicationId,
          applicationLineId: applicationLineId,
          complete: false
        })

Reason being that if we were using Typescript, for example, it wouldn't compile, and given our absence of compile-time type checking, I think that the interface is clearer and the code is more maintainable if we pass in all the properties, even if their values are undefined.

Reply:
Changed as requested.